src/components/Uploader/style.css

```diff
@@ -154,6 +154,7 @@
   }
 }
 
+.contentWrapper,
 .uploaderWrapper {
```

Comment:
Is `uploaderWrapper` being used anywhere else, can we remove it?

Reply:
It's being used on the non-passport document upload screen so I've left it as is.